x-pack/legacy/plugins/security/server/lib/saved_objects_client/secure_saved_objects_client_wrapper.js

```diff
@@ -105,6 +106,26 @@ export class SecureSavedObjectsClientWrapper {
     return await this._baseClient.update(type, id, attributes, options);
   }
 
+  async bulkUpdate(objects = []) {
+    const undefinedNamespace = Symbol('no-namespace');
+    await Promise.all(
```

Comment:
As discussed, we'll simplify this to expect a single namespace for the entire operation, rather than a namespace-per-object

Reply:
Done :)